mix.lock

```diff
@@ -2,24 +2,28 @@
   "cowboy": {:hex, :cowboy, "1.0.0"},
   "cowlib": {:hex, :cowlib, "1.0.1"},
   "decimal": {:hex, :decimal, "1.1.0"},
-  "ecto": {:hex, :ecto, "0.9.0"},
+  "ecto": {:hex, :ecto, "0.12.0-rc"},
   "eredis": {:git, "git://github.com/wooga/eredis.git", "a2273335de773209fd3ab30542fd1b670ca5c957", [tag: 'v1.0.5']},
-  "excoveralls": {:hex, :excoveralls, "0.3.9"},
+  "excoveralls": {:hex, :excoveralls, "0.3.10"},
   "exjsx": {:hex, :exjsx, "3.1.0"},
-  "exq": {:git, "git://github.com/akira/exq.git", "2c1cdc731611e3ee1689c984333edc9d114952a3", []},
+  "exq": {:git, "git://github.com/akira/exq.git", "4fe65d7de524fc7720ee1962fa4de75c33fc7210", []},
   "hackney": {:hex, :hackney, "1.1.0"},
   "idna": {:hex, :idna, "1.0.2"},
+  "inflex": {:hex, :inflex, "0.2.9"},
+  "ja_serializer": {:git, "git://github.com/AgilionApps/ja_serializers.git", "372d3e0d036c2fe04e0b7ef678d1d292299f22c7", []},
+  "ja_serializers": {:git, "git://github.com/AgilionApps/ja_serializers.git", "3bdbb70c670e2d35a75748e0b04d7b3293548407", []},
```

Comment:
Looks like when I renamed the ja_serializer repo to a singular name I caused this.